packages/arb-token-bridge-ui/src/util/networks.ts

```diff
@@ -334,7 +408,10 @@ export function getNetworkName(chainId: number) {
   }
 }
 
-export function getNetworkLogo(chainId: number) {
+export function getNetworkLogo(
+  chainId: number,
+  variant: 'light' | 'dark' = 'dark'
```

Comment:
We need to have a theme-agnostic variant for the Orbit logo. Otherwise, it will be buggy - in mobile, we will need to activate the dark-mode image which is just an overhead for now. We should think of a better solution when we're focused on theming.

<img width="420" alt="image" src="https://github.com/OffchainLabs/arbitrum-token-bridge/assets/7558499/b6860219-bf43-4496-9230-d52ecf669ca8">

Reply:
Having a gray logo will help us get rid of the variant logic for logos.

Reply:
We have a white one, I will set it here

Reply:
https://github.com/OffchainLabs/arbitrum-token-bridge/issues/1110